### ISSUE TITLE
GML file import error is not displayed in Report List

### ISSUE THREAD

**Issue by JohannaOtt** [halestudio/hale#543]
When importing GML files to hale connect, if you get "GML file import"-errors (at least for "java.lang.IllegalArgumentException: Invalid number of points in LinearRing (found 3 - must be 0 or >= 4)") in the Error Log, they are not displayed in the Report List and can thereby easily stay undetected. 
![image](https://user-images.githubusercontent.com/34163838/37289154-47bb6448-2609-11e8-9908-a097612f27e0.png)
![image](https://user-images.githubusercontent.com/34163838/37289194-5d17ed3e-2609-11e8-9ef6-6b674949bfd1.png)

**Comment by JohannaOtt** [comment]
The same is true for warnings. When importing data where no FeatureType is fitting to the FeatureTypes of the available source schema, the Report List does not give a respective information to the user. There is only this warning in the error log:
![grafik](https://user-images.githubusercontent.com/34163838/101477040-f07eb480-394e-11eb-90f7-50f0fc4b4f42.png)
In order to inform users about why no data is occurring in the schema explorer after importing such a "non-matching" dataset, this warning should be displayed in the Report List as well.

**Comment by Kate-Lyndegaard** [comment]
@JohannaOtt Do you have the example GML file used in this issue description?

**Comment by JohannaOtt** [comment]
No, that is too long ago, sorry. Reading the issue, it should be possible to reproduce it by loading a GML file with errors by for example manipulating the coordinates though.
